openapi.yaml

```diff
@@ -5173,6 +5230,7 @@ paths:
     get:
       tags:
         - Users
+      x-sdk-operation: listWorkspace
```

Comment:
Similar to Activities, Users can belong to the root workspace (no `workspaces` in path), or a non-root workspace. This endpoint lists users for a workspace.